jenkins/docker/Dockerfile

```diff
@@ -132,7 +132,7 @@ ENV MVN mvn --batch-mode
 RUN cd /tmp && \
   git clone https://github.com/google/protobuf.git && \
   cd protobuf && \
-  git reset bf379715c93b581eeb078cec1f0dd8a7d79df431 && \
+  git reset 46ae90dc5e145b12fffa7e053a908a9f3e066286 && \
```

Comment:
This line is actually shared by both java and php. php also needs to download github repository.